gitlab_test.go

```diff
@@ -44,7 +45,13 @@ func setup(t *testing.T) (*http.ServeMux, *httptest.Server, *Client) {
 	server := httptest.NewServer(mux)
 
 	// client is the Gitlab client being tested.
-	client, err := NewClient("", WithBaseURL(server.URL))
+	client, err := NewClient("",
+		WithBaseURL(server.URL),
+		// Disable backoff to speed up tests that expect errors.
+		WithCustomBackoff(func(_, _ time.Duration, _ int, _ *http.Response) time.Duration {
+			return 0
```

Comment:
Maybe do `1 * time.Second` or `500 * time.Millisecond` so it at least is measurable/noticeable just in case anyone needs or expects that?

Reply:
I would prefer not to add delay without a counterexample. I didn't see any tests relying on timing. The retry tests that we have rely on the number of times the handler is called. But, if you're certain, then I'd suggest we set the slow tests to be disabled by default, since it's an annoyance when you're doing TDD.